### ISSUE TITLE
Built in commands "addrole" and "removerole" bug

### ISSUE THREAD

**Issue by xDp64xG** [Cog-Creators/Red-DiscordBot#1333]
###Version of Red is Version 3

Please be sure to read through other issues as well to make sure what you are suggesting/reporting has not already
been suggested/reported

### Type:

- [ ] Bug

### Brief description of the problem
The problem is with the [p]addrole and [p]removerole, both don't default to the author if no user is listed. 
The cog was not added by me, one of the base cogs, "admin", to locate the problem.
### Expected behavior
When I do 
[p]addrole Member
I expect it to add the role to me, as it says in the description "If user is left blank it defaults to the author of the command." Same thing applies to [p]removerole as well.
### Actual behavior
What actually happens is an error:
```[21/02/2018 16:13] ERROR events on_command_error 165: Exception in command 'removerole'
Traceback (most recent call last):
  File "C:\Program Files (x86)\Python36-32\lib\site-packages\discord\ext\commands\core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "C:\Program Files (x86)\Python36-32\Lib\site-packages\redbot\cogs\admin\admin.py", line 160, in removerole
    await self._removerole(ctx, user, rolename)
  File "C:\Program Files (x86)\Python36-32\Lib\site-packages\redbot\cogs\admin\admin.py", line 121, in _removerole
    await member.remove_roles(role)
AttributeError: 'NoneType' object has no attribute 'remove_roles'
```
Meaning that you have to mention a user for it to work, meaning you have to mention yourself as well. 
### Steps to reproduce

1. Create a role
2. Try to add the role through the command, [p]addrole <role-name>
3. Try to remove the role through the command, [p]removerole <role-name>
4. Both give errors.

**Comment by unseenmagik** [comment]
![2018-02-22 07_22_22- cmds - discord](https://user-images.githubusercontent.com/30931287/36525348-7da0c6aa-17a1-11e8-9a6f-f43f76bf9d0e.png)
Confirming that @tekulvw changes in #1336 Fix this issue..

**Comment by Kowlin** [comment]
Fixed in #1336
